src/openvic-simulation/politics/Issue.cpp

```diff
@@ -101,9 +119,22 @@ bool IssueManager::_load_reform_group(
 	return ret;
 }
 
-bool IssueManager::_load_reform(size_t& ordinal, std::string_view identifier, ReformGroup const* group, ast::NodeCPtr node) {
-	// TODO: conditions to allow, policy modifiers, policy rule changes
-	return add_reform(identifier, group, ordinal);
+bool IssueManager::_load_reform(
+	ModifierManager& modifier_manager, RuleManager& rule_manager, size_t& ordinal, std::string_view identifier,
```

Comment:
`ordinal` doesn't need to be a reference here, it can just be a regular `size_t`